src/handlers/store-raw-invoice-textract-data/handler.ts

```diff
@@ -0,0 +1,14 @@
+export const handler = async (): Promise<string | undefined> => {
```

Comment:
Could we take in the event and console log it to make it easier for the next ticket that  builds out the function?

Reply:
i see logging the event is in the extract handler (which I'm working on today). Did you mean to add the log to this handler?